homeassistant/helpers/aiohttp_client.py

```diff
@@ -275,31 +286,43 @@ def _async_close_websession(event: Event) -> None:
     hass.bus.async_listen_once(EVENT_HOMEASSISTANT_CLOSE, _async_close_websession)
 
 
+@callback
+def _make_key(verify_ssl: bool = True, family: int = 0) -> tuple[bool, int]:
+    """Make a key for connector or session pool."""
+    return (verify_ssl, family)
```

Comment:
I made this flexible so we can add more combinations later without having to add a lot more complexity

Reply:
May be you could use a named tuple?
https://docs.python.org/3/library/collections.html#collections.namedtuple

Reply:
Since we never access the data in the tuple as its only used for the hash key, I'm not sure if that would be worth the additional complexity.